Sources/MapboxDirections/AdministrativeRegion.swift

```diff
@@ -2,9 +2,9 @@ import Foundation
 
 /**
  :nodoc:
- `AdministrationRegion` describes corresponding object on the route.
+ `AdministrativeRegion` describes corresponding object on the route.
  */
-public struct AdministrationRegion: Codable, Equatable {
+public struct AdministrativeRegion: Codable, Equatable {
 
     private enum CodingKeys: String, CodingKey {
```

Comment:
While you’re at it, can you take a look at #491? I think the fix lies somewhere in here too.

Reply:
#492 takes care of #491.